rosa_backing_up_and_restoring_applications/backing-up-applications.adoc

```diff
@@ -6,6 +6,38 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can employ OpenShift API for Data Protection (OADP) with Red Hat OpenShift Service on AWS (ROSA) clusters to backup and restore application data. A ROSA deployment of OpenShift is configured specifically for AWS services.
+You can employ OpenShift API for Data Protection (OADP) with Red Hat OpenShift Service on AWS (ROSA) clusters to backup and restore application data. Before installing OADP, you must set up role and policy credentials for OADP so that it can use the AWS API. 
 
-include::modules/oadp-installing-oadp-rosa-sts.adoc[leveloffset=+1]
\ No newline at end of file
+This is a two stage process:
+
+. Prepare AWS credentials.
+. Install the OADP Operator and provide it with the IAM role.
+
+include::modules/oadp-preparing-aws-credentials.adoc[leveloffset=+1]
+
+include::modules/oadp-installing-oadp-rosa-sts.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="oadp-addtl-resources_{context}"]
+.Additional resources
+
+* xref:../rosa_backing_up_and_restoring_applications/backing-up-applications.adoc#oadp-preparing-aws-credentials_rosa-backing-up-applications[Preparing AWS credentials]
+
+[id="rosa-backing-up-applications-known-issues"]
+== Known issues
+.Restic is not supported or recommended
+
+* link:https://issues.redhat.com/browse/OADP-1054[CloudStorage: openshift-adp-controller-manager crashloop seg fault with Restic enabled]
+* link:https://issues.redhat.com/browse/OADP-1057[Cloudstorage API: CSI Backup of an app with internal images partially fails with plugin panicked error]
+* (Affects OADP 1.1.x_ only): link:https://issues.redhat.com/browse/OADP-1055[CloudStorage: bucket is removed on CS CR delete, although it doesn't have "oadp.openshift.io/cloudstorage-delete": "true"]
+
+[role="_additional-resources"]
+[id="additional-resources_rosa-backing-up-applications"]
+== Additional resources
+
+* link:https://docs.openshift.com/rosa/rosa_architecture/rosa-understanding.html[Understanding ROSA with STS]
+* link:https://docs.openshift.com/rosa/rosa_getting_started/rosa-sts-getting-started-workflow.html[Getting started with ROSA STS]
+* link:https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html[Creating a ROSA cluster with STS]
+* link:https://docs.openshift.com/container-platform/4.14/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.html[About installing OADP]
+* link:https://docs.openshift.com/container-platform/4.14/storage/container_storage_interface/persistent-storage-csi.html[Configuring CSI volumes]
```

Comment:
Should these two be hardcoded to openshift 4.14?

Reply:
will check but these were always like this on the doc.

Reply:
reverted to 4.13 for now, will follow up to bump back to 4.14 for the main and 4.14 branches